examples/two-columns-layout/src/components/Icons.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx h */
+import { h } from 'preact';
+
+type IconProps = {
+  className?: string;
+};
+
+export const ChevronRightIcon = ({ className }: IconProps) => (
+  <div className={['aa-ItemIcon aa-ItemIcon--noBorder', className].join(' ')}>
+    <svg viewBox="0 0 24 24" fill="none" stroke="currentColor">
```

Comment:
Applicable to the other icons.

```suggestion
    <svg viewBox="0 0 24 24" fill="none" stroke="currentColor" {...props}>
```